pkg/ecdsa/dkg/marshaling_test.go

```diff
@@ -0,0 +1,219 @@
+package dkg
+
+import (
+	fuzz "github.com/google/gofuzz"
+	"github.com/keep-network/keep-core/pkg/crypto/ephemeral"
+	"github.com/keep-network/keep-core/pkg/internal/pbutils"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"reflect"
+	"testing"
+)
+
+func TestEphemeralPublicKeyMessage_MarshalingRoundtrip(t *testing.T) {
+	keyPair1, err := ephemeral.GenerateKeyPair()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	keyPair2, err := ephemeral.GenerateKeyPair()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	publicKeys := make(map[group.MemberIndex]*ephemeral.PublicKey)
+	publicKeys[group.MemberIndex(211)] = keyPair1.PublicKey
+	publicKeys[group.MemberIndex(19)] = keyPair2.PublicKey
+
+	msg := &ephemeralPublicKeyMessage{
+		senderID:            group.MemberIndex(38),
+		ephemeralPublicKeys: publicKeys,
+	}
+	unmarshaled := &ephemeralPublicKeyMessage{}
+
+	err = pbutils.RoundTrip(msg, unmarshaled)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !reflect.DeepEqual(msg, unmarshaled) {
+		t.Fatalf("unexpected content of unmarshaled message")
```

Comment:
non-blocking

Do you think it's worth adding more detail to the failure message? We could print out the sender id and public keys of both messages, for instance

Reply:
Ideally, we can print all fields of both messages. However, in practice, marshaling/unmarshaling errors often require a debugging session anyway. Printing the content may help sometimes but I think this is not worth the additional complexity related to formatting the failure message in a readable way and so on.